src/ServiceProviders/Onboarding.php

```diff
@@ -7,6 +7,7 @@
 use Give\Onboarding\SettingsRepositoryFactory;
 use Give\Onboarding\Setup\Page as SetupPage;
 use Give\Onboarding\Wizard\Page as WizardPage;
+use Give\Onboarding\Wizard\FormPreview as FormPreview;
```

Comment:
The use of `as` isn't necessary here, since it is the same as the class name.

Reply:
Good catch! Missed this in some refactoring. Done, ref: https://github.com/impress-org/givewp/commit/af37134f5b1db7df4897ca4d84afcadc8d77e3a2